bucket/sublime-text.json

```diff
@@ -8,27 +8,61 @@
     },
     "architecture": {
         "64bit": {
-            "url": "https://download.sublimetext.com/Sublime%20Text%20Build%203211%20x64.zip",
-            "hash": "fb77731fa3eda907620a9e8b85bdccc5b5cb4dcf655dcfdc6c7591b78abd0f31"
+            "url": [
+                "https://download.sublimetext.com/Sublime%20Text%20Build%203211%20x64.zip",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/sublime-text/install-context.reg",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/sublime-text/uninstall-context.reg"
+            ],
+            "hash": [
+                "fb77731fa3eda907620a9e8b85bdccc5b5cb4dcf655dcfdc6c7591b78abd0f31",
+                "45914fc3b299e90d9e3c5c84a4c3747c942918462e8b18348ec43383b87ed810",
+                "0253faa4f5e35be203aefa838594965e43aa97129e305d1b8ee1811098e0ae85"
+            ]
         },
         "32bit": {
-            "url": "https://download.sublimetext.com/Sublime%20Text%20Build%203211.zip",
-            "hash": "e169691d88498fdc8dddc72d336680339707d459d12acf665e830f5ed4a4c98d"
+            "url": [
+                "https://download.sublimetext.com/Sublime%20Text%20Build%203211.zip",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/sublime-text/install-context.reg",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/sublime-text/uninstall-context.reg"
+            ],
+            "hash": [
+                "e169691d88498fdc8dddc72d336680339707d459d12acf665e830f5ed4a4c98d",
+                "45914fc3b299e90d9e3c5c84a4c3747c942918462e8b18348ec43383b87ed810",
+                "0253faa4f5e35be203aefa838594965e43aa97129e305d1b8ee1811098e0ae85"
+            ]
         }
     },
+    "pre_install": [
+        "$file = \"$dir\\sublime-text-install-context.reg\"",
+        "if (Test-Path $file) {",
+        "    $sublimepath = \"$dir\\sublime_text.exe\".Replace('\\', '\\\\')",
+        "    $content = Get-Content $file",
+        "    $content = $content.Replace('$sublime', $sublimepath)",
+        "    if ($global) { $content = $content.Replace('HKEY_CURRENT_USER', 'HKEY_LOCAL_MACHINE') }",
+        "    Set-Content $file $content -Encoding Ascii -Force",
+        "}",
+        "$file = \"$dir\\sublime-text-uninstall-context.reg\"",
```

Comment:
Rename

Reply:
Sorry for my carelessness.
Just a little bit nervious about my second pr in github